pkgs/applications/audio/hybridreverb2/default.nix

```diff
@@ -22,7 +22,8 @@
 
 let
   pname = "HybridReverb2";
-  version = "2.1.2";
+  version = "2.1.2+date=2021-12-19";
```

Comment:
Wait, we're now defaulting to this instead of `2.1.2-unstable-2021-12-19`?

Reply:
Changed to `2.1.2-unstable-2021-12-19` according to https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md